docs/lessons/direction.rst

```diff
@@ -0,0 +1,27 @@
+Direction
+---------
+
+There is a compass on the BBC micro:bit. If you ever make a weather station
+use the device to work out the wind direction.
+
+Compass
++++++++
+
+It can also tell you the direction of North like this::
+
+    from microbit import *
+
+    compass.calibrate()
+
+    while True:
+        needle = ((15 - compass.heading()) // 30) % 12
```

Comment:
The modulo and floor division operators have not really been mentioned here or anywhere else. While maybe that's outside of the scope for this documentation, it just feels a bit incomplete having these type of calculation here without any explanation.